.gitignore

```diff
@@ -1,6 +1,6 @@
 # Proto files
-*_pb2.py
-*_pb2_grpc.py
+dlrover/*_pb2.py
+dlrover/*_pb2_grpc.py
```

Comment:
Please make sure you want to keep the Python files generated by atorch pb here? It's generally not to keep auto-generated code.